digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -175,48 +175,51 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
 
     }
     withDynTransaction {
```

Comment:
After fixture reset its normal to have the project "ProjectOne" with road number 77997 and road part 1 which is in insert_project_link_data.sql.  Project_link in DB should be the same before and after   SBT tests